modules/user/user.gql

```diff
@@ -1,13 +1,19 @@
 extend type Query {
     userGetPoolBalances(chains: [GqlChain!], address: String): [GqlUserPoolBalance!]!
     userGetStaking(chains: [GqlChain!], address: String): [GqlPoolStaking!]!
+    """
+    Will de deprecated in favor of poolGetEvents
+    """
     userGetPoolJoinExits(
         first: Int = 10
         skip: Int = 0
         poolId: String!
         chain: GqlChain
         address: String
     ): [GqlPoolJoinExit!]!
+    """
+    Will de deprecated in favor of poolGetEvents
+    """
```

Comment:
could already add the new queries `userGetPoolJoinExitEvents` and `userGetSwapEvents`

Reply:
afais, there is no need for separate queries and we can serve all these use-cases with just one getPoolEvents query. not sure about using namespace neither.

Reply:
true, it's just a matter of how to use the query and namespacing, sometimes easier for the consumer to find it. But I'm open, can also wait